Mathlib/Data/Int/Order/Basic.lean

```diff
@@ -56,6 +56,8 @@ theorem abs_eq_natAbs : ∀ a : ℤ, |a| = natAbs a
   | -[_+1] => abs_of_nonpos <| le_of_lt <| negSucc_lt_zero _
 #align int.abs_eq_nat_abs Int.abs_eq_natAbs
 
+@[simp, norm_cast] lemma coe_natAbs (n : ℤ) : (n.natAbs : ℤ) = |n| := n.abs_eq_natAbs.symm
```

Comment:
Shouldn't this get an `#align`?
```suggestion
@[simp, norm_cast] lemma coe_natAbs (n : ℤ) : (n.natAbs : ℤ) = |n| := n.abs_eq_natAbs.symm
#align int.coe_nat_abs
```

Reply:
How does that work here? The lemma is not in mathlib yet.

Reply:
The two PRs should get merged basically simultaneously, right? And in any case `#align` itself doesn't seem to care about either of the declarations existing, only mathport.

Reply:
@jcommelin  told me that #align fails if the mathlib4 definition doesn't exist; that certainly feels like it *ought* to be the case, but are you claiming it's not?

Reply:
Indeed it cares about the mathlib4 definition existing.